src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
```

Comment:
Idk, something like like...

```js
const worker = new Worker('./my-happy-little-worker.js');
bugsnag.attachBreadcrumbs(worker); // would setup subsequent events/method overrides
worker.postMessage('dance 💃'); // auto-breadcrumb
```

The thing is, going back to the `console.log` issue. If any of these methods (`postMessage` or `terminate`) errored, the stacktrace should still be valid, but the console would just show the top frame as coming from bugsnag.js. Which is arguably not as terrible - but I do understand it's still not favourable. We get this already with `setTimeout` and `requestAnimationFrame` anyway.